integration/pom.xml

```diff
@@ -911,6 +911,15 @@
                 <module>mediation-tests/tests-platform/tests-userstore</module>
             </modules>
         </profile>
+        <profile>
+            <id>transaction-tests</id>
```

Comment:
let's add to db-tests profile as well, to run in Jenkins.

Reply:
Added.